help/en/manual/DecoderPro/index.shtml

```diff
@@ -753,7 +576,7 @@ The Basic Programmer</h4>
 
 </ul>
 
-<h4><a name="Error"></a>Error Messages</h4>
+<h4 name="Error" id="Error">Error Messages</h4>
```

Comment:
This needs to follow the <h4><a name="Error" id="Error"></a> form in order to be fully compatible with the HTML markup required by JavaHelp